variables/_svgs.pcss

```diff
@@ -292,6 +292,13 @@
 
 @svg-load recurring url('../icons/recurring-events.svg') {
 
+	path {
+		fill: var(--color-icon-active);
```

Comment:
Changing this will affect all uses of this variable downstream. Make this consistent with the featured icon, such that `recurring` uses the accent primary color and `recurring-active` uses the icon active color.

Reply:
Yes, the icon color should be the one I used. And in terms of naming, it makes sense to use the `-active` suffix to name the icon for when it is active, at least to me. Or what's was the purpose of the `-active` suffix when you defined the variable names?

I think it wouldn't be to friendly to have `something-active` for the default state and then when it's active `something`.

Reply:
Let's chat about this.

Reply:
Hey guys! I'm not even sure I follow what y'all are discussing, mind filling me in?

Reply:
I'm working on making the recurring icon work as the designs. This is part (with other PRs) of the task. I've modified the `recurring` var with the default state (black) and added the `recurring-active` with the accent color.

Example:
https://www.loom.com/share/0e0a579d86f143a79ccf46e906a1d50d

Paul is stating that, in favor of consistency, we should keep this like what we have for example for the `featured` var where the color is the other way round (`featured` has the accent color, and `featured-active` the default color).

Reply:
We chatted about it and given that these are just variable names, and they'll later be used in classes that can give more context, I'll keep consistency. I'm still not 100% convinced because I find it counterintuitive, mostly thinking of some other devs doing things with this later.

Reply:
Gotcha, so in checking this out, I had to clear my eyes a few times and think about this as we are working with 4 different "var" names here which means it's highly confusing 😂 After understanding what is happening here, I think, the points you are both making are 100% the same and in alignment. We are saying that:

- `recurring` = `--color-accent-primary`
- `recurring-active` = `--color-icon-active`

And we just need to flip those vars around to achieve that 👍

Reply:
😂 You mean flipping, like this?

- `recurring` = `--color-icon-active`
- `recurring-active` = `--color-accent-primary`

If so: that was my initial approach and what triggered the discussion (because the other icons have the color values in a different order)

https://github.com/moderntribe/tribe-common-styles/pull/54/commits/e1b7c93328ddb94fc8d815af70c15abbc768f410

Reply:
I'm so confused haha. It makes sense that the active color var would be synced up with the recurring active svg icon. Isn't that what you are also saying?

> And in terms of naming, it makes sense to use the -active suffix to name the icon for when it is active, at least to me. Or what's was the purpose of the -active suffix when you defined the variable names?

Are the recurring icon names the ones that need to get updated instead based on where they ultimately get used? :P

Reply:
My preference, which you can both ignore, would be that we have a `--color-icon` & `--color-icon-active` var, which would make this clearer 😂